setup.py

```diff
@@ -0,0 +1,36 @@
+import os
+from setuptools import setup
+
+
+def read(filename):
+    return open(os.path.join(os.path.dirname(__file__), filename)).read()
+
+test_requires = [
+    "check-manifest>=0.25",
+    "coverage>=4.0",
+    "isort>=4.2.2",
+    "pytest-cache>=1.0",
+    "pytest-cov>=1.8.0",
+    "pytest-pep8>=1.0.6",
+    "pytest>=3.0.3",
+    "mock>=2.0.0",
+]
+
+setup(
+    name ='yaml2rst',
+    version = '1.0',
+    author = 'Antonio Cesarano',
+    author_email = 'cesarano2607@gmail.com',
+    description = ('Parser for yaml/json schemas to rst'),
+    license = 'BSD',
+    keywords = 'yaml rst parser',
+    url = 'https://github.com/ammirate',
+    packages=['yaml', 'yaml2rst'],
```

Comment:
After correcting the issue https://github.com/inspirehep/yaml2rst/pull/1#pullrequestreview-26274167 and installing with `pip install -e .`, import fails:
```python
In [1]: import yaml2rst
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-83f0b622f873> in <module>()
----> 1 import yaml2rst

/home/micha/src/yaml2rst/yaml2rst/__init__.py in <module>()
     23 # as an Intergovernmental Organization or submit itself to any jurisdiction.
     24 
---> 25 from yaml2rst import yaml2rst
     26 
     27 __all__ = ['yaml2rst.py']

/home/micha/src/yaml2rst/yaml2rst/yaml2rst.py in <module>()
     32 import shutil
     33 
---> 34 from rst_util import NL, RST_DIRECTIVES
     35 from rst_writer import restify, index
     36 from tree_node import TreeNode

ImportError: No module named rst_util
```